setup.py

```diff
@@ -30,14 +30,18 @@
 import shutil
 from setuptools import Extension, Command, setup, find_packages
 
+
+INSTALL_REQUIRES = ["numpy<2.0.0", "scipy<=1.13.1", "tqdm", "powerlaw"]
+
 try:
     from Cython.Distutils import build_ext
     import numpy as np
     import scipy
 except ImportError:
+    escape_dependency_version = lambda x: '"{}"'.format(x) if "<" in x or "=" in x or ">" in x else x
     exit(
         "We need some dependencies to build Cornac.\n"
-        + "Run: pip3 install Cython numpy scipy"
+        + "Run: pip3 install Cython {}".format(" ".join([escape_dependency_version(x) for x in INSTALL_REQUIRES]))
```

Comment:
@tqtg Just updated the usage of lambda function. There is some packages with specified versions. The output of the suggested command looks like this `Run: pip3 install Cython "numpy<2.0.0" "scipy<=1.13.1" tqdm powerlaw`